Initial XR support (#3245)
f3d-app/f3d: doc/libf3d/03-OPTIONS.md

### `render.grid.absolute` (_bool_, default: `false`)

> Position the grid at the _absolute origin_ of the model's coordinate system instead of below the model. Forced to true if XR mode is enabled.

Reviewer: ```suggestion
Position the grid at the _absolute origin_ of the model's coordinate system instead of below the model. Forced to true with `xr` rendering backend.
```

Reviewer: generalize that everywhere you mention "XR mode"
